### PR TITLE
(maint) Revert a bunch of stuff

### PR DESCRIPTION
**Pull request by mwaggett**
Apparently this broke some things. Reverting for now while I figure out the issue.

### CODE REVIEW

**Comment by mwaggett** [comment]
https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_nightly-repo_packaging_ship/3352/ is still finishing up, but is past the point of previous failure.

File: tasks/nightly_repos.rake [hunk 1]
```diff
@@ -42,9 +42,9 @@ DOC
       Pkg::Rpm::Repo.create_local_repos('repos')
       Pkg::Rpm::Repo.sign_repos('repos')
       Pkg::Deb::Repo.sign_repos('repos', 'Apt repository for signed builds')
-      Pkg::Sign::Dmg.sign('repos') unless Dir['repos/apple/**/*.dmg'].empty?
-      Pkg::Sign::Ips.sign('repos') unless Dir['repos/solaris/11/**/*.p5p'].empty?
-      Pkg::Sign::Msi.sign('repos') unless Dir['repos/windows/**/*.msi'].empty?
+      Pkg::Sign.sign_osx('repos') unless Dir['repos/apple/**/*.dmg'].empty?
+      Pkg::Sign.sign_ips('repos') unless Dir['repos/solaris/11/**/*.p5p'].empty?
+      Pkg::Sign.sign_msi('repos') unless Dir['repos/windows/**/*.msi'].empty?
```
Comment (underscorgan) [inline tasks/nightly_repos.rake]: Doesn't this part also depend on a revert of the reorg of all the signing stuff?

Comment (mwaggett) [reply tasks/nightly_repos.rake]: You would think.. but somehow it still passed.. https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_nightly-repo_packaging_ship/3352/consoleFull

Comment (underscorgan) [reply tasks/nightly_repos.rake]: I want to figure out why this is working, this doesn't make sense.

Comment (underscorgan) [reply tasks/nightly_repos.rake]: Those methods don't exist in your fork of packaging and so I'm worried it's getting them from some other place. Can we please either un-revert this or revert both this and the refactor?

**Comment by mwaggett** [comment]
@underscorgan unreverted. Started a new job to test: https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_nightly-repo_packaging_ship/3353/

**Comment by mwaggett** [comment]
welp I guess that commit is part of the problem..

**Comment by mwaggett** [comment]
@underscorgan I reverted associated refactoring, but turns out my testing isn't great because it's still pulling in packaging#1.0.x (instead of my branch) via bundle install.

**Comment by underscorgan** [comment]
How are you getting this to use your branch? Is there a way we can hand-modify a clone of the job briefly to get a better test?

**Comment by mwaggett** [comment]
@underscorgan I was hand-modifying the nightly ship job to clone my branch of packaging instead of bootstrapping. but that doesn't work for remote stuff, so my testing was not.. complete/thorough/accurate. :/

I can build a new puppet-agent pointing to my branch, though.

**Comment by mwaggett** [comment]
@underscorgan Built this version of p-a from my branch, job is still running, but past previous failure points (just repo updates rn): https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_nightly-repo_packaging_ship/

**Comment by underscorgan** [comment]
the job output still looks like it's using packaging from 1.0.x
```
15:35:48 Using packaging 0.99.2.59.g1efea59 from git://github.com/puppetlabs/packaging.git (at 1.0.x@1efea59)
```

**Comment by mwaggett** [comment]
I don't know what's happening.
If anyone has ideas, please let me know. But right now, I feel like we should get this in, to unblock things, and then figure out what's going on bit by bit.

**Comment by underscorgan** [comment]
@mwaggett how did you build your agent (which branch were you building off of, was it on your fork)?

**Comment by mwaggett** [comment]
@underscorgan http://jenkins-release.delivery.puppetlabs.net/job/vanagon_generic_job/1424/ (didn't care that AIX failed) - I pushed a branch from my fork of puppet-agent (https://github.com/mwaggett/puppet-agent/tree/test) that pointed to my branch from my fork of packaging.

**Comment by underscorgan** [comment]
@mwaggett you also need to update the packaging version in your Gemfile to point to your branch, that's what's getting used for the installs.

**Comment by mwaggett** [comment]
(╯°□°）╯︵ ┻━┻

**Comment by mwaggett** [comment]
@underscorgan It's gonna take almost 3 hours to rebuild..

**Comment by underscorgan** [comment]
Usually I'm not quite this careful with reverts, but since it's reverting a subset of commits from bigger refactoring PRs it seemed like it would be good to have some confirmation that we didn't miss anything 🤷‍♀️

**Comment by mwaggett** [comment]
@underscorgan I get that, but it's already broken.. 
Started a new build (with a subset of platforms) though.

**Comment by mwaggett** [comment]
OKAY https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_nightly-repo_packaging_ship/3360/ is building for-realsies with my branch of packaging and looking good.

**Pull request merged**